packages/block-editor/src/components/block-bindings-toolbar-indicator/index.js

```diff
@@ -22,13 +22,10 @@ import useBlockDisplayTitle from '../block-title/use-block-display-title';
 
 export default function BlockBindingsToolbarIndicator( { clientIds } ) {
```

Comment:
If I am not mistaken, this component becomes specific to pattern overrides because it is only used when adding a pattern in a post, right? Maybe we can rename it and slightly change the implementation to reflect that?

Reply:
On second thought, I don't think this even belongs in the `block-editor` package 😅. I'd imagine it'll be a bigger refactor if we figure out a way to move it to the `patterns` package though. I'd prefer to do it in a follow-up PR if necessary. WDYT?

Reply:
Moving it to the `patterns` package seems logical to me 🙂 And I guess it is okay to do it in a follow-up PR. Should we create an issue or something to ensure it is not lost?

Maybe it is still worth changing the component name to clarify it is related to patterns and not block bindings until that PR is done?

Reply:
Done in [c702ddb](https://github.com/WordPress/gutenberg/pull/62317/commits/c702ddb66e0cb8b919e23a84164891b3f56d6991)! I renamed some stuff, so could appreciate a final review to ensure I didn't break anything 😆.